extensions/wikia/ArticleVideo/OoyalaBacklotApiService.class.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class OoyalaBacklotApiService {
+	private $api;
+
+	public function __construct() {
+		global $wgOoyalaApiConfig;
+
+		$this->api = new OoyalaApi(
+			$wgOoyalaApiConfig['AppId'],
+			$wgOoyalaApiConfig['AppKey']
+		);
+	}
+
+	//TODO use memcache
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)